src/ruleLoader.ts

```diff
@@ -123,18 +122,18 @@ function transformName(name: string) {
  * @param directory - An absolute path to a directory of rules
  * @param ruleName - A name of a rule in filename format. ex) "someLintRule"
  */
-function loadRule(directory: string, ruleName: string) {
+function loadRule(directory: string, ruleName: string): RuleConstructor | null {
     const fullPath = path.join(directory, ruleName);
     if (fs.existsSync(fullPath + ".js")) {
-        const ruleModule = require(fullPath);
-        if (ruleModule && ruleModule.Rule) {
+        const ruleModule = require(fullPath) as { Rule: RuleConstructor } | undefined;
+        if (ruleModule !== undefined) {
             return ruleModule.Rule;
         }
     }
-    return undefined;
+    return null;
```

Comment:
maybe we should use a string literal like "not-found" or "load-error" instead of `null`